src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -59,6 +59,7 @@ export type UpdateQueue = {
   first: Update | null,
   last: Update | null,
   hasForceUpdate: boolean,
+  callbackList: null | Array<Callback>,
```

Comment:
In context it is a fixed general stack. In normal use it never gets reallocated or expanded. The wasm implementation can be as easy as a fixed one time linear allocation. Where as this is dynamically allocated and growing.